app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -28,19 +28,35 @@ function initializeCommentDropdown() {
     }
   }
 
-  function copyText() {
+  function copyPermalink(event) {
+    event.preventDefault();
+    const permalink = event.target.href;
+
+    Runtime.copyToClipboard(permalink)
+      .then(() => {
+        // eslint-disable-next-line no-undef
+        addSnackbarItem({ message: 'Copied to clipboard' });
+      })
+      .catch((e) => {
+        console.log(e);
```

Comment:
Refactor (blocking): This catch is not necessary because if this fails, it's an unexpected exception and Honeybadger (our reporting tool), will not be able to log it as the `console.log` will swallow the error.

By default, Honeybadger handles unhandledrejections. See https://github.com/honeybadger-io/honeybadger-js/blob/bc074bc335164d40655489d017bfcd9f0f64a36d/src/browser/integrations/onunhandledrejection.ts#L9

Reply:
Alright, thanks for that note, I've removed the catch statement. :+1: 
I'll keep this comment unresolved so that it'll be easier to notice when checking the changes.
Please let me know if there's anything else :smile: